### PR TITLE
Remove tera-input-number. Only use tera-input

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Note
This may be a breaking change for many things!!! it is important to test to triple check that your stuff is not broken
I have done a quick check on all of these fields and will do another but please take a minute to review

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue [hunk 1]
```diff
@@ -53,8 +54,9 @@
 		<div class="input-row">
 			<div class="label-and-input">
 				<label for="lower-bound">Lower bound</label>
-				<tera-input-number
+				<tera-input
 					class="p-inputtext-sm"
+					type="number"
 					:min-fraction-digits="1"
 					:max-fraction-digits="10"
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue:61]: these aren't valid props

File: packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue [hunk 2]
```diff
@@ -27,7 +27,8 @@
 											{{ id }}
 										</td>
 										<td>
-											<tera-input-number
+											<tera-input
+												type="decimal"
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue:31]: This isn't a valid type is it @blanchco ?

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue:31]: Decimal seemed to work in a very quick test

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue:31]: inputmode?: 'none' | 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search';

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue (hunk 1 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-policy-group-form.vue:61]: You are correct. Im not sure if we want to add this functionality in the future to our input or what so i left them

File: packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue (hunk 2 above)
Comment (blanchco) [reply packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue:31]: decimal should be fine...we may push the text to the right like we do with number though

**Review by shawnyama — APPROVED** [review]

**Comment by Tom-Szendrey** [comment]
I need to ensure I remove a lot of class="..." and whatnot before I merge

**Review by Tom-Szendrey — COMMENTED** [review]
I need to ensure I remove a lot of class="..." and whatnot before I merge

**Pull request merged**